src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!--  <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="frame">
+        <legend>
+          Personal information
+        </legend>
+        <label class="field">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        </legend>
        
        <label class="field">
```